homeassistant/components/netatmo/light.py

```diff
@@ -0,0 +1,146 @@
+"""Support for the Netatmo camera lights."""
+import logging
+
+import pyatmo
+
+from homeassistant.components.light import LightEntity
+from homeassistant.core import callback
+from homeassistant.exceptions import PlatformNotReady
+
+from .const import DATA_HANDLER, DOMAIN, MANUFACTURER
+from .data_handler import NetatmoDataHandler
+from .netatmo_entity_base import NetatmoBase
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Netatmo camera light platform."""
+    if "access_camera" not in entry.data["token"]["scope"]:
+        _LOGGER.info(
+            "Cameras are currently not supported with this authentication method"
+        )
+        return
+
+    data_handler = hass.data[DOMAIN][entry.entry_id][DATA_HANDLER]
+
+    data_class_name = "CameraData"
+
+    async def get_entities():
+        """Retrieve Netatmo entities."""
+        await data_handler.register_data_class(data_class_name)
```

Comment:
```suggestion
    async def get_entities():
        """Retrieve Netatmo entities."""
        await data_handler.register_data_class(CAMERA_DATA_CLASS_NAME)
```

Maybe we can get rid of the magic strings here by making these constant that are imported from .data_handler

and then update `DATA_CLASSES` in `.date_handler` to use them as well.